src/helpers.js

```diff
@@ -276,6 +276,15 @@ const transform = {
         up: _("up"),
         down: _("down"),
     },
+    watchdogAction: {
+        reset: _("Reset"),
+        shutdown: _("Gracefully shutdown"),
+        poweroff: _("Power off"),
+        pause: _("Pause"),
+        none: _("Do nothing"),
+        dump: _("Dump"),
+        "inject-nmi": _("Non-maskable interupt"),
```

Comment:
How about use the libivrt docs wording? 

https://libvirt.org/formatdomain.html#watchdog-device

'Inject a non-maskable interrupt into the guest'

Reply:
Seems a bit too long. and looks too long in the UI compared to the others:
![Screenshot from 2023-01-09 23-58-09](https://user-images.githubusercontent.com/42733240/211425112-57ae6b09-83cd-4a6b-80a5-2d900bc26cfa.png)
How about just saying
"Inject a non-maskable interrupt"? I guess that the interrupt happens on the VMs level can be safely assumed.